.github/CODEOWNERS

```diff
@@ -6,19 +6,22 @@
 *                   @DataDog/agent-integrations
 
 # Documentation
-/docs/              @DataDog/baklava @DataDog/agent-integrations
-*.md                @DataDog/baklava @DataDog/agent-integrations
-manifest.json       @DataDog/baklava @DataDog/agent-integrations
+/docs/                @DataDog/baklava @DataDog/agent-integrations
+*.md                  @DataDog/baklava @DataDog/agent-integrations
+manifest.json         @DataDog/baklava @DataDog/agent-integrations
 
 # Container monitoring
-/docker_daemon/     @DataDog/container-integrations @DataDog/agent-integrations
-/ecs_fargate/       @DataDog/container-integrations @DataDog/agent-integrations
-/kube_dns/          @DataDog/container-integrations @DataDog/agent-integrations
-/kube_proxy/        @DataDog/container-integrations @DataDog/agent-integrations
-/kubelet/           @DataDog/container-integrations @DataDog/agent-integrations
-/kubernetes/        @DataDog/container-integrations @DataDog/agent-integrations
-/kubernetes_state/  @DataDog/container-integrations @DataDog/agent-integrations
+/docker_daemon/       @DataDog/container-integrations @DataDog/agent-integrations
+/ecs_fargate/         @DataDog/container-integrations @DataDog/agent-integrations
+/kube_dns/            @DataDog/container-integrations @DataDog/agent-integrations
+/kube_proxy/          @DataDog/container-integrations @DataDog/agent-integrations
+/kubelet/             @DataDog/container-integrations @DataDog/agent-integrations
+/kubernetes/          @DataDog/container-integrations @DataDog/agent-integrations
+/kubernetes_state/    @DataDog/container-integrations @DataDog/agent-integrations
+
+# Agent core team
+/datadog_checks_base/ @DataDog/agent-core @DataDog/agent-integrations
```

Comment:
This is the added line, everything else are indentation changes

Reply:
Could we be added to `datadog_checks_base/datadog_checks/base/checks/openmetrics` & `datadog_checks_base/datadog_checks/base/checks/prometheus`? 🙏

Reply:
Actually changes to the base checks might affect Docker builds as well, would it make sense to add you to `datadog_checks_base`?